concrete/src/Page/Sitemap/Element/SitemapPage.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace Concrete\Core\Page\Sitemap\Element;
+
+use Concrete\Core\Error\UserMessageException;
+use Concrete\Core\Page\Page;
+use Concrete\Core\Url\UrlInterface;
+use DateTime;
+use SimpleXMLElement;
+
+class SitemapPage extends SitemapElement
+{
+    /**
+     * The page.
+     *
+     * @var \Concrete\Core\Page\Page
+     */
+    protected $page;
+
+    /**
+     * The URL of the page.
+     *
+     * @var \Concrete\Core\Url\UrlInterface
+     */
+    protected $url;
+
+    /**
+     * The last modification date/time.
+     *
+     * @var \DateTime|null
+     */
+    protected $lastMod;
+
+    /**
+     * The page change frequency.
+     *
+     * @var string
+     */
+    protected $changeFrequency;
+
+    /**
+     * The page priority.
+     *
+     * @var string
+     */
+    protected $priority;
+
+    /**
+     * The pages in alternative languages mapped to this page.
+     *
+     * @var \Concrete\Core\Page\Sitemap\Element\SitemapPageAlternativeLanguage[]
+     */
+    protected $alternativeLanguages = [];
+
+    /**
+     * Should this page be skipped?
+     *
+     * @var bool
+     */
+    protected $skip = false;
+
+    /**
+     * @param Page $page the page
+     * @param UrlInterface $url the URL of the page
+     * @param \DateTime|null the last modification date/time
+     * @param string $changeFrequency the page change frequency
+     * @param string $priority the page priority
+     * @param null|DateTime $lastMod
```

Comment:
this one should be removed / changed, I think? (see line 65)

Reply:
:+1; (php-cs-fixed added it for me because I forgot to add the parameter name at line 64)